packages/fetchai/skills/generic_seller/behaviours.py

```diff
@@ -148,8 +148,7 @@ def register_service_personality_classification(self) -> None:
         strategy = cast(GenericStrategy, self.context.strategy)
         descriptions = [
             strategy.get_register_service_description(),
-            strategy.get_register_personality_description(),
-            strategy.get_register_classification_description(),
```

Comment:
I wasn't aware we already set these. So in some sense your PR isn't needed..it just offers a different way of doing it, which is still valuable in general.

Reply:
We do in some skills, not all. 

So task 1 is to combine the two registration steps, and task 2 is to essentially copy paste this solution into all skills.